container-core/src/test/java/com/yahoo/container/di/ContainerTestBase.java

```diff
@@ -31,20 +30,14 @@ public class ContainerTestBase {
 
     @BeforeEach
     public void setup() {
-        dirConfigSource = new DirConfigSource(tmpDir, "ContainerTest-");
+        dirConfigSource = new DirConfigSource(tmpDir);
         componentGraph = new ComponentGraph(0);
         osgi = new TestOsgi(BundleTestUtil.testBundles());
     }
 
-    @AfterEach
-    public void cleanup() {
-        dirConfigSource.cleanup();
```

Comment:
How is cleanup done ?

Reply:
The @TempDir takes care of it. The cleanup that was done here did nothing but return `false` (indicating it failed to delete the non-empty directory).